snippets/language-asciidoc.cson

```diff
@@ -44,10 +44,10 @@
       ----
       $0
       '''
-  'Code Block':
-    description: 'Code Block'
+  'Source Block':
+    description: 'Source block'
     descriptionMoreURL: 'http://asciidoctor.org/docs/user-manual/#listing-blocks'
-    prefix: '--s'
+    prefix: '``'
```

Comment:
in the markdown language the prefix is `code`, use the same if you want but I don't like, I prefer `--s`